dandi/dandiapi.py

```diff
@@ -1071,19 +1073,24 @@ def iter_upload_raw_asset(
             self, metadata=asset_metadata, jobs=jobs, replacing=replace_asset
         )
 
-    def is_nwb(self) -> bool:
+    def has_data_standard(self, data_standard: str) -> bool:
         """
-        Returns True if the Dandiset contains one or more NWB file assets.
-
-        This is determined by checking for "RRID:SCR_015242" in the "dataStandard" field
-        of the assetsSummary of the dandiset.
+        Returns True if the Dandiset contains one or more files of the indicated
+        standard. Otherwise, returns False.
```

Comment:
The docstring should document the accepted data standards (or, at the very least, the RRID input format).